services/api/src/app.js

```diff
@@ -40,7 +40,7 @@ app.on('error', (err, ctx) => {
   }
   // dont output stacktraces of errors that is throw with status as they are known
   if (!err.status || err.status === 500) {
-    ctx.logger.error(err);
+    logger.error(err);
```

Comment:
no ctx.logger ... not that a problem. just want to ensure that error is generated in the right context (e.g. the tracing is working).

not sure if the ctx.logger ensure that or not?

Reply:
also you might be missing https://github.com/bedrockio/bedrock-core/blob/88eb9b5e7c18273e4def2b9d4ce651ec8ff85576/services/api/src/utils/middleware/cors.js#L24

Reply:
It should work as the context is supplied by the tracer context adding the spanId but I'll have a look into the CORS thing I might have missed that.